controllers/gitlabidentityprovider/controller.go

```diff
@@ -107,6 +107,8 @@ func (r *Controller) ReconcileDelete(req controllers.Request) (ctrl.Result, erro
 }
 
 // SetupWithManager sets up the controller with the Manager.
+//
+//nolint:wrapcheck
```

Comment:
Why are we disabling wrapcheck here?

Reply:
This is scaffolded from Kubebuilder.  Wrapcheck is a pretty opinionated check IMO.  I disable this frequently when wrapping errors doesn't make sense.